### PR TITLE
Add support for new Wavelet (Himitsu)

### LINKED ISSUE #58: First load results in blank screen
Currently, on the Himitsu branch, when you open lens for the first time it results in a blank blue screen.
2019-12-06-120043_1919x978_scrot

Console has a couple of "Max call stack exceeded" errors: 
```
2.45465cb8.chunk.js:1 [mobx] Encountered an uncaught exception that was thrown by a reaction or observer component, in: 'Reaction[Reaction@8]' RangeError: Maximum call stack size exceeded
    at L (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
    at t.get [as notification] (2.45465cb8.chunk.js:1)
A.reportExceptionInDerivation @ 2.45465cb8.chunk.js:1
A.track @ 2.45465cb8.chunk.js:1
v @ 2.45465cb8.chunk.js:1
(anonymous) @ 2.45465cb8.chunk.js:1
A.runReaction @ 2.45465cb8.chunk.js:1
...
(anonymous) @ main.e33f414b.chunk.js:1
Show 6 more frames
2.45465cb8.chunk.js:1 RangeError: Maximum call stack size exceeded
    at L (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
vo @ 2.45465cb8.chunk.js:1
So.t.callback @ 2.45465cb8.chunk.js:1
xi @ 2.45465cb8.chunk.js:1
_i @ 2.45465cb8.chunk.js:1
Da @ 2.45465cb8.chunk.js:1
e.unstable_runWithPriority @ 2.45465cb8.chunk.js:1
...
(anonymous) @ main.e33f414b.chunk.js:1
2.45465cb8.chunk.js:1 Uncaught RangeError: Maximum call stack size exceeded
    at L (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
    at t.get [as api] (2.45465cb8.chunk.js:1)
```

### LINKED ISSUE #59: When sending the exact balance in a user account, the transaction silently fails
If an account has 1 PERL, and you send exactly 1 PERL, you will get a "successfully sent TX" message, but the TX will actually fail & deduct the TX fee, leaving the user with 0.999999998 PERLs

### LINKED ISSUE #60: Lens tx graph breaks the chain when switching between pages
Screenshot 2019-12-06 at 15 27 21

### PR DESCRIPTION
**Pull request by claudiucelfilip**
Major changes add to support Wavelet (Himitsu):
- adapted partial transaction graph functionality. Now each node represents a block.
- updated transaction table and details info based on new Wavelet responses
- updated wavelet-client to v2
- added minimum of 100 KENs for reward withdrawal, similar to Wavelet restriction
- fixed #58 
- fixed #59 
- fixed #60 

Branch was build on top of new Create React App. (PR #55 )

### CODE REVIEW

File: tsconfig-backuo.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (ryanswart) [inline tsconfig-backuo.json]: What's up with this file? Can't we use git history?

Comment (claudiucelfilip) [reply tsconfig-backuo.json]: It's no longer needed. Removed it.

File: package.json [hunk 2]
```diff
@@ -7,189 +7,101 @@
...
         "tslint-config-prettier": "^1.10.0",
         "tslint-react": "^3.2.0",
         "tweetnacl": "^1.0.0",
-        "uglifyjs-webpack-plugin": "^1.1.8",
-        "url-loader": "0.6.2",
-        "vis": "^4.21.0",
+        "typescript": "3.7.2",
         "wabt": "^1.0.10",
-        "wavelet-client": "^1.1.0",
+        "wavelet-client": "^2.0.0",
```
Comment (ryanswart) [inline package.json]: Can we test with the rc published version

Comment (claudiucelfilip) [reply package.json]: Changed with rc2

**Pull request merged**